helm_deploy/laa-access-civil-legal-aid/templates/ingress.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
   labels:
     {{- include "laa-access-civil-legal-aid.labels" . | nindent 4 }}
   annotations:
+    nginx.ingress.kubernetes.io/custom-http-errors: "413,502,504"
```

Comment:
Can you clarify my understanding of this please.
Does this forward 413's, 502's and 504's to Cloud Platform's default Ngninx error page?